docs/hardware/antenna/antenna-report.mdx

```diff
@@ -10,4 +10,4 @@ sidebar_position: 2
 
 ### Meshtastic-Antenna-Reports
 
-RichInNewMexico has gone throught the trouble of testing a number of commonly purchaed antennas in the meshtastic community and given an opinion on weather or not a given antenna is worth using. please check out the project at [Meshtastic-Antenna-Reports](https://github.com/RicInNewMexico/Meshtastic-Antenna-Reports)
+RicInNewMexico has gone through the trouble of testing a number of commonly purchased antennas in the Meshtastic community and given an opinion on whether or not a given antenna is performing optimally. please check out the project at [Meshtastic-Antenna-Reports](https://github.com/RicInNewMexico/Meshtastic-Antenna-Reports)
```

Comment:
I noticed this a little late. If we get `please` capitalized, I'll get this merged in.